dev/com.ibm.ws.cdi.2.0_fat/.classpath

```diff
@@ -1,8 +1,37 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
 	<classpathentry kind="src" path="fat/src"/>
+	<classpathentry kind="src" path="test-applications/interceptionFactoryApp/src"/>
 	<classpathentry kind="src" path="test-applications/beanManagerLookupApp/src"/>
 	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="lib" path="/com.ibm.websphere.javaee.interceptor.1.2/generated/com.ibm.websphere.javaee.interceptor.1.2.jar">
+		<attributes>
+			<attribute name="bsn" value="com.ibm.websphere.javaee.interceptor.1.2.jar"/>
+			<attribute name="type" value="PROJECT"/>
+			<attribute name="project" value="com.ibm.websphere.javaee.interceptor.1.2"/>
+			<attribute name="version" value="latest"/>
+			<attribute name="index_location" value="platform:/plugin/bndtools.builder/org/bndtools/builder/classpath/empty.index"/>
+		</attributes>
+		<accessrules>
+			<accessrule kind="accessible" pattern="javax/interceptor/*"/>
+			<accessrule ignoreifbetter="true" kind="discouraged" pattern="**"/>
+		</accessrules>
+	</classpathentry>
+	<classpathentry kind="lib" path="/com.ibm.websphere.javaee.annotation.1.2/generated/com.ibm.websphere.javaee.annotation.1.2.jar">
+		<attributes>
+			<attribute name="bsn" value="com.ibm.websphere.javaee.annotation.1.2.jar"/>
+			<attribute name="type" value="PROJECT"/>
+			<attribute name="project" value="com.ibm.websphere.javaee.annotation.1.2"/>
+			<attribute name="version" value="latest"/>
+			<attribute name="index_location" value="platform:/plugin/bndtools.builder/org/bndtools/builder/classpath/empty.index"/>
+		</attributes>
+		<accessrules>
+			<accessrule kind="accessible" pattern="javax/annotation/*"/>
+			<accessrule kind="accessible" pattern="javax/annotation/security/*"/>
+			<accessrule kind="accessible" pattern="javax/annotation/sql/*"/>
+			<accessrule ignoreifbetter="true" kind="discouraged" pattern="**"/>
+		</accessrules>
+	</classpathentry>
```

Comment:
I don't think you should have libs in the .classpath. I think the api dependencies should all be done via the bnd file.

Reply:
Yes. That change to .classpath should not be in the commit